src/supporting-data/database/config/DatabaseColumnConfiguration.tsx

```diff
@@ -0,0 +1,121 @@
+import { Link } from 'react-router-dom';
+import { ExternalLink, LongNumber } from 'franklin-sites';
+
+import EntryTypeIcon from '../../../shared/components/entry/EntryTypeIcon';
+
+import { getEntryPathFor } from '../../../app/config/urls';
+
+import { DatabaseAPIModel } from '../adapters/databaseConverter';
+import { ColumnConfiguration } from '../../../shared/types/columnConfiguration';
+import { EntryType } from '../../../uniprotkb/adapters/uniProtkbConverter';
+import { Namespace } from '../../../shared/types/namespaces';
+
+export enum DatabaseColumn {
+  abbrev = 'abbrev',
+  category = 'category',
+  // URL template for a specific entry within this database
+  dbUrl = 'dbUrl',
+  doiId = 'doi_id',
+  id = 'id',
+  linkType = 'link_type',
+  name = 'name',
+  pubmedId = 'pubmed_id',
+  reviewedProteinCount = 'reviewed_protein_count',
+  // URL of the home page of the database
+  server = 'server',
+  unreviewedProteinCount = 'unreviewed_protein_count',
+}
+
+// TODO: review
+export const defaultColumns = [
+  DatabaseColumn.id,
+  DatabaseColumn.name,
+  DatabaseColumn.abbrev,
+  DatabaseColumn.reviewedProteinCount,
+  DatabaseColumn.unreviewedProteinCount,
+  DatabaseColumn.category,
+];
+
+export const primaryKeyColumn = DatabaseColumn.id;
+
+const getEntryPath = getEntryPathFor(Namespace.database);
+
+export const DatabaseColumnConfiguration: ColumnConfiguration<
+  DatabaseColumn,
+  Partial<DatabaseAPIModel>
+> = new Map();
+
+// COLUMN RENDERERS BELOW
+DatabaseColumnConfiguration.set(DatabaseColumn.abbrev, {
+  label: 'Abbreviation',
+  render: ({ abbrev }) => abbrev,
+});
+
+DatabaseColumnConfiguration.set(DatabaseColumn.category, {
+  label: 'Category',
+  render: ({ category }) => category,
+});
+
+DatabaseColumnConfiguration.set(DatabaseColumn.dbUrl, {
+  label: 'DB URL',
+  render: ({ dbUrl }) => dbUrl,
```

Comment:
I don't think it is for website users, but maybe for API/download users? If so it would still needs to be there at least to be exposed in the download dropdown.
Wondering if there could be a better way to show this is not a link. Could change the column label and wrap the URL pattern in a CodeBlock for example?

Reply:
Using CodeBlock sounds like a good improvement. Regarding the label I wonder what would be the best description of the data. It's a template string that when filled in and accessed is used to get a particular entry from that database so `DB Entry URL Template`?

Reply:
Changed